### PR TITLE
[QA-1935] Terra-UI test log: Capture page loading requests from ui server

### PR DESCRIPTION
**Pull request by aweng98**
https://broadworkbench.atlassian.net/browse/QA-1935

### CODE REVIEW

File: integration-tests/utils/integration-utils.js [hunk 1]
```diff
@@ -365,7 +365,8 @@ const logPageAjaxResponses = page => {
   const terraRequests = [
     'broad',
     'terra',
-    'googleapis'
+    'googleapis',
+    'bvdp'
```
Comment (aweng98) [inline integration-tests/utils/integration-utils.js:369]: Missed it before, but we should also capture `bvdp-*` UI server requests and write them to test log.

**Review by kyuksel — APPROVED** [review]

File: integration-tests/utils/integration-utils.js (hunk 1 above)
Comment (nawatts) [reply integration-tests/utils/integration-utils.js:369]: What requests are made to `bvdp-*`? So far as I can tell, it's only the request to load `config.json`?

Comment (aweng98) [reply integration-tests/utils/integration-utils.js:369]: Static files. Example screenshot for project: `bvdp-saturn-staging`.

Screen Shot 2022-06-30 at 10 51 51 AM

Comment (nawatts) [reply integration-tests/utils/integration-utils.js:369]: Ah, I see. I was misled by the name of this function (`logPageAjaxResponses`). Requests for static files are not AJAX requests. But it looks like the `response` event is not specific to AJAX requests.
https://pptr.dev/#?product=Puppeteer&version=v13.5.1&show=api-event-response

**Review by nawatts — APPROVED** [review]
Maybe rename `logPageAjaxResponses` since this will now also log responses for non-AJAX requests for static files.

**Comment by aweng98** [comment]
Good idea. Renamed function to `logPageResponses`.

**Pull request merged**